### ISSUE TITLE
ShellCheck should be disabled on json-array style RUN instructions

### ISSUE THREAD

**Issue by iphydf** [hadolint/hadolint#418]
- [x] This is a bug report
- [x] I searched existing issues before opening this one

### Expected behavior

No warning.

### Actual behavior

* Double quote to prevent globbing and word splitting.
* d is referenced but not assigned.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Hadolint 1.17.5
```

**Dockerfile**

```Dockerfile
RUN ["sed", "-i", "/^bootstrap_nodes = /,$d", "other/bootstrap_daemon/tox-bootstrapd.conf"]
```

**Comment by m-ildefons** [comment]
Fixed in v2.15.0
